js/levels.js

```diff
@@ -17,7 +17,7 @@ function createFirstLevel()
     [door.drawnbyme, Superhero.art, PlayerStartsAt(door.black), door.green, door.banner, door.threeHeads],
     [door.staircase, door.highLow, door.new, door.wheel, door.plain, door.top],
     [door.fairysWheel, door.design, door.cricketGround, door.treasure, door.yellow, door.red],
-    [door.both, door.top, door.marina, door.treasureKey, door.drawn, door.top],
+    [door.both, door.top, door.marina, door.treasureKey, door.drawn, door.nighttop],
```

Comment:
this is "nighttop"

Reply:
Sorry, I thought I have to add door name there. It's ok now. Thank you!